gdal/swig/include/cpl.i

```diff
@@ -520,6 +521,27 @@ void wrapper_VSIFileFromMemBuffer( const char* utf8_path, int nBytes, const GByt
 #endif
 #endif
 
+#if defined(SWIGJAVA)
+%inline %{
+void wrapper_VSIFileFromMemBuffer( const char* utf8_path, void* nioBuffer, long nioBufferSize) {
```

Comment:
This method is already available in the Java case a bit above. Are we sure that the 2 can co-exist ? Adding a test program in swig/java/apps/ checking  that both work would be needed